cli/src/commands/git/push.rs

```diff
@@ -172,6 +177,13 @@ pub struct GitPushArgs {
         add = ArgValueCandidates::new(complete::mutable_revisions)
     )]
     change: Vec<RevisionArg>,
+    /// Specify a new bookmark name and a revision to push under that name, e.g.
+    /// '--named myfeature=@'.
+    ///
+    /// Does not require --allow-new
```

Comment:
nit: in clap doc string, remove period from the summary line, and add period to detailed paragraphs.